src/commit.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+echo "::group::Commit"
+
+cd "$INPUT_PACKAGE_NAME"
+
+echo "Push the new PKGBUILD and .SRCINFO files to the AUR repo"
+git add PKGBUILD .SRCINFO
+commit "$(generate_commit_message "" "$NEW_RELEASE")"
+git push
+
+if [[ "$INPUT_UPDATE_PKGBUILD" != "true" || -z "$INPUT_AUR_SUBMODULE_PATH" ]]; then
```

Comment:
Here it should be `&&` and not `||` because we want it to skip this step if `INPUT_UPDATE_PKGBUILD` is not `true` and at the same time the user has not provided us any value in `INPUT_AUR_SUBMODULE_PATH`